app/templates/skeleton/index.html

```diff
@@ -3,7 +3,7 @@
 <head>
   <title></title>
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-	<meta charset="utf-8">
+  <meta charset="utf-8">
```

Comment:
Roses are red, violets are blue, first indentations use spaces next should too :wink: